onnx/defs/tensor/defs.cc

```diff
@@ -1794,4 +1794,132 @@ ONNX_OPERATOR_SET_SCHEMA(
 
           propagateShapeFromInputToOutput(ctx, 0, 0);
         }));
+
+static const char* Unique_ver11_doc = R"DOC(
+Find the unique elements of a tensor. When an optional attribute 'axis' is provided, unique subtensors along the 'axis' are returned. 
+Otherwise the input tensor is flattened and unique values of the flattened tensor are returned. 
+
+This operator returns the unique elements of the input tensor and three optional outputs. 
+The first output tensor 'Y' contains all unique values or subtensors of the input. 
+The second optional output tensor 'indices' contains indices of the the unique elements in input. 
```

Comment:
How about "The second optional output tensor 'indices': indices[j] contains the index of the first occurrence of Y[j] in input X."

Reply:
Good one! Updated the doc.